routes/trips.js

```diff
@@ -13,6 +13,7 @@ router.use(decodeUserFromToken)
 router.post('/', checkAuth, tripCtrl.create)
 router.get('/', checkAuth, tripCtrl.index)
 router.get('/:id', checkAuth, tripCtrl.show)
+router.put('/addToTrip', checkAuth,tripCtrl.createActivityPlan)
```

Comment:
@mrunlockedtech-odin I already added an activities route to create an activityPlan.  We don't need this right?  lets get aligned....